codec_specs.md

```diff
@@ -14,44 +14,99 @@ Each of these elements contain the same kind of data. And these data depend on t
+  
+**Codec ID:** V_MPEG4/ISO/AP  
+**Codec Name:** MPEG4 ISO advanced profile  
+**Description:** (Same as above)
+
+  
+**Codec ID:** V_MPEG4/MS/V3  
+**Codec Name:** Microsoft (TM) MPEG4 V3  
+**Description:** and derivates, means DivX3, Angelpotion, SMR, etc.; stream was created using VfW codec or transmuxed from AVI; note that V1/V2 are covered in VfW compatibility mode
+
+  
+**Codec ID:** V_MPEG1  
+**Codec Name:** MPEG 1  
+**Description:** The matroska video stream will contain a demuxed Elementary Stream (ES ), where block boundaries are still to be defined. Its RECOMMENDED to use MPEG2MKV.exe for creating those files, and to compare the results with selfmade implementations
+
+  
+**Codec ID:** V_MPEG2  
+**Codec Name:** MPEG 2  
+**Description:** The matroska video stream will contain a demuxed Elementary Stream (ES ), where block boundaries are still to be defined. Its RECOMMENDED to use MPEG2MKV.exe for creating those files, and to compare the results with selfmade implementations
+
+  
+**Codec ID:** V_REAL/????  
+**Codec Name:** Real Video(TM)   
+**Description:** The stream is one of the Real Video(TM) video streams listed below. Source for the codec names are from [Karl Lillevold on Doom9](http://forum.doom9.org/showthread.php?s=&amp;threadid=55773&amp;perpage=20&amp;pagenumber=2#post331855). The CodecPrivate element contains a "real_video_props_t" structure in Big Endian byte order as found in [librmff](https://github.com/mbunkus/mkvtoolnix/blob/master/lib/librmff/librmff.h).
+
+  
+**Codec ID:** V_REAL/RV10  
+**Codec Name:** RealVideo 1.0 aka RealVideo 5  
+**Description:** Individual slices from the Real container are combined into a single frame.
+
+  
+**Codec ID:** V_REAL/RV20  
+**Codec Name:** RealVideo G2 and RealVideo G2+SVT  
+**Description:** Individual slices from the Real container are combined into a single frame.
+
+  
+**Codec ID:** V_REAL/RV30  
+**Codec Name:** RealVideo 8  
+**Description:** Individual slices from the Real container are combined into a single frame.
+
+  
+**Codec ID:** V_REAL/RV40  
+**Codec Name:** rv40 : RealVideo 9  
+**Description:** Individual slices from the Real container are combined into a single frame.
+
+  
+**Codec ID:** V_QUICKTIME  
+**Codec Name:** Video taken from QuickTime(TM) files  
+**Description:** Several codecs as stored in QuickTime, e.g. Sorenson or Cinepak. The CodecPrivate contains all additional data that is stored in the 'stsd' (sample description) atom in the QuickTime file **after** the mandatory video descriptor structure (starting with the size and FourCC fields). For an explanation of the QuickTime file format read [QuickTime File Format Specification](https://developer.apple.com/library/mac/documentation/QuickTime/QTFF/QTFFPreface/qtffPreface.html).
+
+  
+**Codec ID:** V_THEORA  
+**Codec Name:** Theora  
+**Description:** The private data contains the first three Theora packets in order. The lengths of the packets precedes them. The actual layout is: 
```

Comment:
not blocking this PR, could resolve the expressions of CodecPrivate in a subsequent PR